app/src/main/java/com/example/playlistmaker/adapter/SearchAdapter.kt

```diff
@@ -0,0 +1,74 @@
+package com.example.playlistmaker.Adapter
+
+import android.content.Context
+import android.content.Intent
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners
+import com.example.playlistmaker.util.MusicHistory
+import com.example.playlistmaker.R
+import com.example.playlistmaker.presentation.SongActivity
+import com.example.playlistmaker.util.Debounce
+import com.example.playlistmaker.model.Track
+import com.example.playlistmaker.model.tracks
+import java.text.SimpleDateFormat
+import java.util.*
+
+class searchAdapter(private val context: Context) : RecyclerView.Adapter<searchAdapter.SearchViewHolder>() {
```

Comment:
🍏 Классы в Kotlin принято называть с большой буквы. https://kotlinlang.org/docs/coding-conventions.html#naming-rules
```suggestion
class SearchAdapter(private val context: Context) : RecyclerView.Adapter<searchAdapter.SearchViewHolder>() {
```